mock-recipe-server/bin/ci/deploy

```diff
@@ -1,4 +1,11 @@
 #!/usr/bin/env bash
-set -eu
+set -e
+
+if [[ -z $MOCK_SERVER_ARTIFACTS || -z $MOCK_SERVER_S3_BUCKET ]]; then
```

Comment:
Do you know what envvars AWS creds go in? Or can I ask the `aws` tool about that?